packages/client/src/modules/blocks/data-blocks/form-editor/EditorFieldsSider.tsx

```
@@ -0,0 +1,822 @@
+                      if (relationInterfaces.includes(initialValue.interface)) {
+                        initialValue.target = '__temp__';
+                        initialValue.targetKey = 'id';
+                      }
+                      initialValue.uiSchema.title = item.title || compile('Unnamed');
+                      if (initialValue.reverseField) {
+                        initialValue.reverseField.name = `f_${uid()}`;
+                      }
+                      if (initialValue.autoCreateReverseField) {
+                        /* empty */
+                      } else {
+                        delete initialValue.reverseField;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)